tap_googleads/tap.py

```diff
@@ -126,6 +127,25 @@ class TapGoogleAds(Tap):
             description="Enables the tap's ClickViewReportStream. This requires setting up / permission on your google ads account(s)",
             default=False,
         ),
+        th.Property(
+            "custom_queries_array",
```

Comment:
I would prefer to not have the container type (`array`) in the setting name (e.g. `tables` from `tap-spreadsheets-anywhere`):

```suggestion
            "custom_queries",
```